src/uncontrast/pages/EffectsDistributionLines/EffectsDistributionLines.jsx

```diff
@@ -62,36 +64,96 @@ export default function EffectsDistributionLines() {
       "distribution_of_effects_across_parameters",
       significance,
       experimentsNum,
+      binSize,
       selected?.value || continuousBreakdownOptions[0].value,
     ],
     queryFn: () =>
       getEffectsDistribution({
         significance,
         min_number_of_experiments: experimentsNum,
+        binSize,
         continuous_breakdown:
           selected?.value || continuousBreakdownOptions[0].value,
         isUncontrast: true,
       }),
   });
 
+  // const trace1 = {
+  //   x: data?.data[0]?.series.map((a) => a.key),
+  //   y: data?.data[0]?.series.map((a) => a.value),
+  //   opacity: 0.5,
+  //   type: "bar",
+  //   marker: { color: "green" },
+  //   name: data?.data[0].series_name,
+  // };
+  // const line1 = {
+  //   x: data?.data[0]?.series.map((a) => a.key),
+  //   y: data?.data[0]?.series.map((a) => a.value),
+  //   opacity: 0.5,
+  //   type: "scatter",
+  //   mode: "lines", // Only draw lines, no markers
+  //   line: {
+  //     shape: "spline", // Makes the line smooth and curvy
+  //   },
+  //   marker: { color: "green" },
+  //   name: data?.data[0].series_name,
+  // };
+  // const trace2 = {
+  //   x: data?.data[1]?.series.map((a) => a.key),
+  //   y: data?.data[1]?.series.map((a) => a.value),
+  //   opacity: 0.5,
+  //   type: "bar",
+  //   marker: { color: "blue" },
+  //   name: data?.data[1]?.series_name,
+  // };
+  // const trace3 = {
+  //   x: data?.data[2]?.series.map((a) => a.key),
+  //   y: data?.data[2]?.series.map((a) => a.value),
+  //   opacity: 0.5,
+  //   type: "bar",
+  //   marker: { color: "red" },
+  //   name: data?.data[2]?.series_name,
+  // };
+  // const graphsData = [trace1, line1, trace2, trace3];
+  const colors = { Positive: "#159DEA", Mixed: "#088515", Negative: "#CA535A" };
   const graphsData = [];
-  data?.data.map((row) => {
-    graphsData.push({
-      x: row.series.map((a) => a.year),
-      y: row.series.map((a) => a.value),
-      type: "scatter",
-      name: rawTextToShow(row.series_name),
-      mode: "lines+markers",
-    });
+  data?.data.map((row, index) => {
+    graphsData.push(
```

Comment:
You're using a map function to iterate through the data, either use map to actually map or use `forEach`. Map is better.